src/tools/similarity.ts

```diff
@@ -99,6 +100,7 @@ export class SimilarityTool<TProviderOptions> extends Tool<
         documentIndex: idx,
         score,
       })),
+      filter(({ score }) => score >= (options.minScore ?? 0)),
```

Comment:
Is the score always positive? Maybe negative infinity would be a safer sentinel here

Reply:
Good catch. Updated.